File: JAVA/CreditCard.java
package JAVA;

class CreditCard extends Payment {
    Integer cvc;
    String date;
    String bank;

    public CreditCard(Integer id, Integer amount, Integer cvc, String date, String bank) {
        super(id, amount);
        this.cvc = cvc;
        this.date = date;
        this.bank = bank;
    }

}

File: PYTHON/CreditCard.py
from payment import Payment

class CreditCard(Payment):
    cvv = int
    date = str
    bank = str
    
    def __init__(self, id, amount, cvv, date, bank):
        super.__init__(id, amount)
        self.cvv = cvv
        self.date = date
        self.bank = bank
        